internal/resource/rabbitmq_plugins.go

```diff
@@ -0,0 +1,97 @@
+package resource
+
+import (
+	"strings"
+
+	rabbitmqv1beta1 "github.com/rabbitmq/cluster-operator/api/v1beta1"
+	"github.com/rabbitmq/cluster-operator/internal/metadata"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var requiredPlugins = []string{
+	"rabbitmq_peer_discovery_k8s", // required for clustering
+	"rabbitmq_prometheus",         // enforce prometheus metrics
+	"rabbitmq_management",
+}
+
+const pluginsConfigMapName = "plugins-conf"
```

Comment:
small: may I suggest to call it "plugins" to shorten it?

Reply:
Do you mean the name of the constant?

Reply:
I will rename it to pluginsConfig to indicate that it's a configuration.

Reply:
```
const pluginsConfigMapName = "plugins-conf"
```
to 
```
const pluginsConfigMapName = "plugins"
```

So that the name of the configMap is shorted